hack/cli-coexistence/clean_machine.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/zsh
+rm -rf $HOME/tanzu/cli        # Remove previously downloaded cli files
+rm -rf $HOME/go/bin/tanzu     # Remove previously downloaded cli files
```

Comment:
You shouldn't need to do this, as long as your PATH is set properly

Reply:
yea Agree, these are current recommended instructions for uninstalling tanzu https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-uninstall.html . We need to revise it when we provide the recommended instructions of uninstalling new tanzu cli